client/templates/app/nav/_nav.sass

```diff
@@ -6,7 +6,7 @@ body.home, body.show-initiative
     box-shadow: none
     
 nav
-  z-index: 1
+  z-index: 999
```

Comment:
I know this is probably just me, but I dislike when we have super-high z-indexes. Like at the time, it always makes sense, but later when you want something overtop it, and then you have to put another super-high z-index to beat the previous one. Maybe just make it as high as it needs to be?